### ISSUE TITLE
Plugin PHP requirement

### ISSUE THREAD

**Issue by fantasticmachine** [workingconcept/cloudflare-craft-plugin#43]
The plugin appears to be locked to PHP 7 and so I  am unable to perform Craft updates when using PHP 8.

Console output:

```
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - workingconcept/craft-cloudflare is locked to version 1.0.1 and an update of this package was not requested.
    - workingconcept/craft-cloudflare 1.0.1 requires php ^7.2.0 -> your php version (8.0.8; overridden via config.platform, same as actual) does not satisfy that requirement.

<warning>Running update with --no-dev does not mean require-dev is ignored, it just means the packages will not be installed. If dev requirements are blocking the update you have to resolve those problems.</warning>
```

From what I understand `^7.2.0` means `>=7.2.0 < 8.0`
I've not yet dug under the hood to see if the plugin would break using PHP 8

**Comment by mattstein** [comment]
Thanks @fantasticmachine, it should probably be fine with PHP 8 but I’ll make sure before I update that requirement.

**Comment by mattstein** [comment]
Just published [1.0.2](https://github.com/workingconcept/cloudflare-craft-plugin/releases/tag/1.0.2) removing the explicit PHP requirement since it isn’t necessary.

The (really simple) tests now include PHP 8 and are passing, and my local testing with PHP 8 didn’t uncover anything scandalous. Let me know if you’re still having any sort of issues!
